lib/views/answer/form/form_with_text_field_view.dart

```diff
@@ -1,10 +1,82 @@
 import 'package:flutter/material.dart';
+import 'package:kayla_flutter_ic/utils/border_radiuses.dart';
+import 'package:kayla_flutter_ic/views/question/answers_container_ui_model.dart';
 
-class FormWithTextFieldView extends StatelessWidget {
-  const FormWithTextFieldView({super.key});
+class FormWithTextFieldView extends StatefulWidget {
+  final List<OptionUiModel> uiModels;
+  final Function(Map<String, String>) onChange;
+
+  const FormWithTextFieldView({
+    super.key,
+    required this.uiModels,
+    required this.onChange,
+  });
+
+  @override
+  State<FormWithTextFieldView> createState() => _FormWithTextFieldViewState();
+}
+
+class _FormWithTextFieldViewState extends State<FormWithTextFieldView> {
+  final List<TextEditingController> _controllers = [];
+  final Map<String, String> answers = <String, String>{};
+
+  @override
+  void dispose() {
+    for (var controller in _controllers) {
+      controller.dispose();
+    }
+    super.dispose();
+  }
 
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return _buildForm();
+  }
+
+  Widget _buildForm() {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: widget.uiModels.map(
+        (uiModel) {
+          final controller = TextEditingController();
```

Comment:
technically, we should create the controllers once in StatefulWidget. So putting the initialization in the `build` method will trigger the controller initialization multiple times. Can you put this in `initState` instead?